app/adapters/agendaitem.js

```diff
@@ -3,11 +3,15 @@ import ApplicationAdapter from './application';
 export default ApplicationAdapter.extend({
   findHasMany (store, snapshot, url, relationship) {
     // Workaround for Ember Data not taking paging into account when fetching a hasMany
+    const namesOnly = snapshot.adapterOptions && snapshot.adapterOptions.namesOnly;
     if (relationship.meta.key === 'documentVersions') {
-      url = url + '?page[size]=9999'
+      url = url + '?page[size]=200';
+      if (namesOnly) {
+        url += '&fields[document-versions]=name';
+      }
     }
     if (relationship.meta.key === 'linkedDocumentVersions') {
-      url = url + '?page[size]=9999'
+      url = url + '?page[size]=200'
```

Comment:
Gaan we niet het risico lopen dat we op een gegeven moment niet alles exporteren, en dan "voelt het weer inconsistent?"